<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="book_green">
    <static>false</static>
    <link name="link">
      <inertial>
        <mass>0.2</mass>
        <inertia>
          <ixx>0.0001</ixx> <ixy>0</ixy> <ixz>0</ixz>
          <iyy>0.0001</iyy> <iyz>0</iyz> <izz>0.0001</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry><box><size>0.03 0.15 0.2</size></box></geometry>
      </collision>
      <visual name="visual">
        <geometry><box><size>0.03 0.15 0.2</size></box></geometry>
        <material>
          <script>
            <uri>file://media/materials/scripts/gazebo.material</uri>
            <name>Gazebo/Green</name>
          </script>
        </material>
      </visual>
      <visual name="qrcode_visual">
        <pose>0.016 0 0.05 0 0 0</pose>
        <geometry><box><size>0.001 0.08 0.08</size></box></geometry>
        <material>
          <script>
            <uri>model://book_tags/materials/scripts</uri>
            <uri>model://book_tags/materials/textures</uri>
            <name>BookTag/ShelfC</name>
          </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>
